### PR TITLE
switch sidebar form pages to block view when someone clicks any block

### PR DESCRIPTION
**Pull request by mb324-unikent**

### CODE REVIEW

File: resources/assets/js/components/Block.vue [hunk 1]
```diff
@@ -146,11 +147,15 @@ export default {
...
 
 		editBlock() {
+			// set the block in the sidebar if the user clicks on a different block (including one with the same index but in a different region)
 			if(this.currentBlockIndex !== this.index || this.currentRegion !== this.region) {
 				this.collapseSidebar();
-				this.updateMenuActive("blocks");
 				this.setBlock({ index: this.index, type: this.type });
 			}
+			// make sure we get to see the block menu if we're currently seeing the pages menu and a user clicks on any block
+			if(this.active==='pages') {
```
Comment (theninja) [inline resources/assets/js/components/Block.vue]: ```this.active !== 'blocks'```?

File: resources/assets/js/components/Block.vue [hunk 2]
```diff
@@ -69,7 +69,8 @@ export default {
 			currentRegion: state => state.page.currentRegion,
 			currentBlockIndex: state => state.definition.currentBlockIndex,
 			draggingBlocks: state => state.page.dragging,
-			allBlockMeta: state => state.page.blockMeta.blocks
+			allBlockMeta: state => state.page.blockMeta.blocks,
+			active: state => state.menu.active
```
Comment (theninja) [inline resources/assets/js/components/Block.vue]: ```activeMenuItem```?

File: resources/assets/js/components/Block.vue (hunk 1 above)
Comment (mb324-unikent) [inline resources/assets/js/components/Block.vue]: done

File: resources/assets/js/components/Block.vue (hunk 2 above)
Comment (mb324-unikent) [inline resources/assets/js/components/Block.vue]: done

**Pull request merged**